libsweep/README.md

```diff
@@ -4,19 +4,36 @@ Low-level Scanse Sweep LiDAR library. Comes as C99 library `sweep.h` with option
 
 ### Quick Start
 
-    mkdir -p build
-    cd build
-    cmake .. -DCMAKE_BUILD_TYPE=Release
-    cmake --build .
-    sudo cmake --build . --target install
-    sudo ldconfig
+```bash
+mkdir -p build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
+cmake --build .
+sudo cmake --build . --target install
+sudo ldconfig
+```
 
 If you don't have a Sweep device yet you can build a dummy `libsweep.so` always returning static point cloud data:
 
-    cmake .. -DCMAKE_BUILD_TYPE=Release -DDUMMY=On
+```bash
+cmake .. -DCMAKE_BUILD_TYPE=Release -DDUMMY=On
+```
 
 This dummy library is API and ABI compatible. Once your device arrives switch out the `libsweep.so` shared library and you're good to go.
 
+For windows users, open a command prompt with administrative access:
+
+```bash
+mkdir build
+cd build
+# The 14 stands for VS 2015
+cmake .. -G "Visual Studio 14 Win64"
+cmake --build . --config Release
```

Comment:
DCMAKE_BUILD_TYPE is specific to make generators (or at least it doesn't work with IDE style generators (Visual Studio, and potentially XCode).  I do believe that `--config Release` won't cause any issues with make style generators... if this can be verified maybe we should switch over to that for unix as well, just to be consistent?